city_scrapers/spiders/il_liquor_control.py

```diff
@@ -0,0 +1,161 @@
+import datetime
+import requests
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlLiquorControlSpider(CityScrapersSpider):
+    name = "il_liquor_control"
+    agency = "Illinois Liquor Control Commission"
+    timezone = "America/Chicago"
+    allowed_domains = ["www2.illinois.gov"]
+    start_urls = [
+        "https://www2.illinois.gov/ilcc/Divisions/Pages/Legal/"
+        "Hearing-Schedule-for-Chicago-IL-and-Springfield-IL.aspx",
+        "https://www2.illinois.gov/ilcc/Divisions/Pages/Legal/Meeting-minutes.aspx"
+    ]  # start_urls[0] contains links future meetings. start_urls[1] contains past
+    # meeting minutes links
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+
+        Each tentative meeting has own page. start_urls[0] is only used to get links for the pages.
+        The links are sent to _next_meeting.
+
+        start_urls[1] is for obtaining datetime info for past meetings. Then checks if
+        meeting detail page is still valid. If valid scrape through _prev_meeting_w_page. If not
+        valid, scrape available info from _prev_meeting_wo_page.
+        """
+
+        if response.url == self.start_urls[0]:
+            for future_meeting_href in response.css('div.soi-link-item a::attr(href)').extract():
+                yield response.follow(future_meeting_href, callback=self._next_meeting)
+
+        if response.url == self.start_urls[1]:
+            for idx, past_meeting_date in enumerate(response.css("div.link-item ::text").extract()):
+                dt_object = datetime.datetime.strptime(past_meeting_date, "%B %d, %Y Minutes")
+                minute_href = response.css("div.link-item a::attr(href)").extract()[idx]
+                meeting_url = ("https://www2.illinois.gov/ilcc/Events/Pages/Board-"
+                               "Meeting-{0}-{1}-{2:%y}.aspx".format(
+                                dt_object.month, dt_object.day, dt_object))
+                if requests.get(meeting_url).status_code == 200:
```

Comment:
An easier way of doing this than requesting the same page twice to check the error code might be to override scrapy's behavior and allow parsing 404 pages. That way, we can handle missing pages in the same method by just setting defaults. You should be able to do this by adding this property under `start_urls` to the spider:

```python
custom_settings = {"HTTPERROR_ALLOW_ALL": True}
```